app/views/deployments.html

```diff
@@ -77,8 +116,11 @@ <h3 ng-if="(deployments | hashSize) || (replicaSets | hashSize)">Deployment Conf
                       <div row class="status">
                         <status-icon status="replicationController | deploymentStatus" disable-animation></status-icon>
                         <span flex>
-                          {{replicationController | deploymentStatus}}<span ng-if="(replicationController | deploymentStatus) == 'Active' || (replicationController | deploymentStatus) == 'Running'">,
-                          <span ng-if="replicationController.spec.replicas !== replicationController.status.replicas">{{replicationController.status.replicas}}/</span>{{replicationController.spec.replicas}} replica<span ng-if="replicationController.spec.replicas != 1">s</span></span>
+                          {{replicationController | deploymentStatus}}
```

Comment:
you have introduced whitespace here that will cause the , to have a space in front of it